deploy/hwameistor/manifest.yaml

```diff
@@ -350,6 +356,9 @@ spec:
             - matchExpressions:
               - key: node-role.kubernetes.io/master
                 operator: Exists
+            - matchExpressions:
+              - key: node-role.kubernetes.io/control-plane
```

Comment:
This key `node-role.kubernetes.io/control-plane` should be on the same level as `node-role.kubernetes.io/master`